spring-integration-smb/src/main/java/org/springframework/integration/smb/filters/SmbSystemMarkerFilePresentFileListFilter.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2012-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.smb.filters;
+
+import jcifs.smb.SmbFile;
+import org.springframework.integration.file.filters.AbstractMarkerFilePresentFileListFilter;
+import org.springframework.integration.file.filters.FileListFilter;
+
+import java.util.Map;
+import java.util.function.Function;
+
+/**
+ * Implementation of {@link AbstractMarkerFilePresentFileListFilter} for SMB.
+ *
+ * @since 0.6.0
+ * @author Prafull Kumar Soni
```

Comment:
`@since` goes after an `@author`.
Anyway we discussed do not `@since` in the current version.